process-engine/bpmn-studio
# ♻️ Use ProcessInstance in InspectCorrelation

**The TokenViewer will be fixed via #1450**

**Changes:**

1. Use ProcessInstance in CorrelationList of InspectCorrelation
2. Use ProcessInstance in DiagramViewer of InspectCorrelation
3. Use ProcessInstance in LogViewer of InspectCorrelation
4. Fix LogTable Sorting When Changing Correlation

PR: #1448

## How can others test the changes?

- Run some processes with the same CorrelationId
- Navigate to the Inspect Correlation
- **Notice that the CorrelationList will have an entry for each processInstance of the active ProcessModel**

## PR-Checklist

Please check the boxes in this list after submitting your PR:
